src/views/Swap.vue

```diff
@@ -203,6 +210,10 @@ export default class Swap extends Mixins(TranslationMixin, LoadingMixin, NumberF
     return isMaxButtonAvailable(this.areTokensSelected, this.tokenFrom, this.fromValue, this.networkFee)
   }
 
+  get isXorAssetUsed (): boolean {
+    return isXorAccountAsset(this.tokenFrom) || isXorAccountAsset(this.tokenTo)
```

Comment:
did you talk with someone from back-end team about the direction of XOR? I mean the case below about liquidity source